xml/System.Diagnostics.CodeAnalysis/SuppressMessageAttribute.xml

```diff
@@ -325,7 +325,7 @@ The <xref:System.Diagnostics.CodeAnalysis.SuppressMessageAttribute.MessageId%2A>
   
 ## Remarks
 
-The <xref:System.Diagnostics.CodeAnalysis.SuppressMessageAttribute.Scope%2A> property is an optional argument that specifies the metadata scope for which the attribute is relevant. Possible values for managed metadata include "member", "resource", "module", "type", "method", or "namespace". For more information about scopes, see [Suppress code analysis warnings](/visualstudio/code-quality/in-source-suppression-overview#suppressmessage-attribute).  
+The <xref:System.Diagnostics.CodeAnalysis.SuppressMessageAttribute.Scope%2A> property is an optional argument that specifies the metadata scope for which the attribute is relevant. Possible values for managed metadata include "member", "resource", "module", "type", "method", "namespace" or "namespaceanddescendants". For more information about scopes, see [Suppress code analysis warnings](/visualstudio/code-quality/in-source-suppression-overview#suppressmessage-attribute).  
```

Comment:
we use oxford commas and probably good to indicate this is new as the VS docs are doing 

```suggestion
The <xref:System.Diagnostics.CodeAnalysis.SuppressMessageAttribute.Scope%2A> property is an optional argument that specifies the metadata scope for which the attribute is relevant. Possible values for managed metadata include "member", "resource", "module", "type", "method", "namespace", or "namespaceanddescendants" (available since Visual Studio 2019). For more information about scopes, see [Suppress code analysis warnings](/visualstudio/code-quality/in-source-suppression-overview#suppressmessage-attribute).  
```